arm9/lib/NitroSDK/include/SPI_mic.h

```diff
@@ -1,6 +1,23 @@
 #ifndef NITRO_SPI_MIC_H_
 #define NITRO_SPI_MIC_H_
 
+#include "global.h"
```

Comment:
including global.h in NitroSDK can cause some weirdness with circular includes, I would generally discourage it